web/template/watch.gohtml

```diff
@@ -116,10 +116,45 @@
                             {{end}}
                         {{end}}
                     {{end}}
-                    <p class="vjs-no-js">
-                        To view this video please enable JavaScript.
-                    </p>
-                </video-js>
+                <p class="vjs-no-js">
+                    To view this video please enable JavaScript.
+                </p>
+            </video-js>
+
+            <div
+                    x-data="{ size: { x: '0px', width: '0px' }, seekLogger: new watch.SeekLogger('{{$stream.ID}}'), heatMapPath: null }"
+                    x-on:updateheatmappath.window="e => { heatMapPath = e.detail.heatMapPath; }"
+                    x-on:updateheatmapsize.window="e => { size = e.detail; }"
+                    x-init="seekLogger.attach(); watch.repeatHeatMap.init('{{$stream.ID}}');"
+                    class="heatmap-wrap absolute bottom-[18px] rounded bg-gradient-to-t from-gray-800/[.6] pt-4 pointer-events-none"
+                    :style="{width: size.width, left: size.x}"
+                    x-show="heatMapPath != null"
+            >
+                <svg x-if="heatMapPath" height="100%" preserveAspectRatio="none" viewBox="0 0 1000 100"
```

Comment:
The `x-if` directive is only allowed on `<template>` tags. (See [this link](https://koenwoortman.com/alpinejs-cannot-read-property-clonenode-of-undefined/) for more information. 

Since currently Alpine throws this error:
<img width="1055" alt="image" src="https://user-images.githubusercontent.com/29633518/210319333-dd756d45-57d2-472e-87fb-aa05dc9d66ae.png">